### PR TITLE
Fix typo error in output of Ballerina By Example

### PR DESCRIPTION
**Pull request by shrikarsan**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=org.ballerinalang
 # During the release workflow, the following will get bumped automatically
-version=2201.2.0-SNAPSHOT
+version=2201.2.0
```
Comment (MaryamZi) [inline gradle.properties:4]: Shall we undo the changes in this file and include just the changes to the out files in this PR?

Comment (shrikarsan) [reply gradle.properties:4]: Yes sure, this change in this file was added by ballerina-bot. I am relatively new to this, could you please undo the changes or tell me what I have to do to undo changes in this file?

Comment (MaryamZi) [reply gradle.properties:4]: You could send a separate PR with just this change. Would that work? You can use the [current master branch of this repo](https://github.com/ballerina-platform/ballerina-distribution) when creating the new branch.

**Comment by MaryamZi** [comment]
Changes merged via https://github.com/ballerina-platform/ballerina-distribution/pull/3227.
